validphys2/src/validphys/results.py

```diff
@@ -1098,6 +1153,53 @@ def fits_chi2_table(
     return res
 
 
+@table
+def fits_nsigma_table(
+    fits_total_chi2_data,
+    fits_datasets_nsigma_table,
+    fits_groups_nsigma_table,
+    show_total: bool = False,
+):
+    """Show the nsigma of each and number of points of each dataset and experiment
+    of each fit, where experiment is a group of datasets according to the `experiment` key in
+    the PLOTTING info file, computed with the theory corresponding to the fit. Dataset that are not
+    included in some fit appear as `NaN`
+    """
+    lvs = fits_groups_nsigma_table.index
+    # The explicit call to list is because pandas gets confused otherwise
+    expanded_index = pd.MultiIndex.from_product((list(lvs), ["Total"]))
+    edf = fits_groups_nsigma_table.set_index(expanded_index)
+    ddf = fits_datasets_nsigma_table
+    dfs = []
+    # TODO: Better way to do the merge preserving the order?
```

Comment:
Why do you want a better way?

(f there's no specific reason, I'd rather remove the `TODO`, I don't really see the problem with what you are doing. If you want leave a comment like

```
# Merge the columns of this and that into a dataframe
```
in case someone wants to have a go in the future)

I'm realizing we have many `TODO` that are really not that important. I mean, there might be a `pandas.take_by_index(edf, lv)` or whatever that together with `pandas.join_magically` does what you want... but the older I get the more I appreciate simple loops and appends that can be ported to some other language or library in the future.

(I'm rambling, I did say I was getting old)